pkg/controllers/virtualmachineinstance_controller.go

```diff
@@ -54,7 +54,9 @@ func (r *VirtualMachineInstanceReconciler) Reconcile(ctx context.Context, reques
 		// Error reading the object - requeue the request.
 		return ctrl.Result{}, err
 	}
-	err = r.ZoneManager.UpdateZone(request.NamespacedName, FilterMultusNonDefaultInterfaces(vmi.Status.Interfaces, vmi.Spec.Networks))
+	filteredInterfaces := FilterMultusNonDefaultInterfaces(vmi.Status.Interfaces, vmi.Spec.Networks)
+	filteredInterfaces = FilterInterfacesWithNoName(vmi.Status.Interfaces) // The interface/network name is used to build the FQDN, therefore, interfaces reported without a name are filtered out
```

Comment:
Aint you override the list that was created in the previous line ?

Reply:
yes! Good catch:) The linter cached it as well.